### PR TITLE
Fix news no stop

### PR DESCRIPTION
**Pull request by ken-mycroft**
## Description
Fixes Jira SKILL-108

## How to test
Try to stop the news playback using anything other than the word stop

## Contributor license agreement signed?
yes

### CODE REVIEW

**Comment by devops-mycroft** [comment]
Voight Kampff Integration Test Succeeded  ([Results](https://reports.mycroft.ai/core/PR-2897))

**Review by krisgesling — APPROVED** [review]
Nice one!

File: mycroft/skills/common_play_skill.py [hunk 1]
```diff
@@ -165,6 +165,8 @@ def __handle_play_start(self, message):
         # "... on the chromecast"
         self.play_service_string = phrase
 
+        self.bus.emit(Message('active_skill_request',  {'skill_id': self.skill_id}))
```
Comment (JarbasAl) [inline mycroft/skills/common_play_skill.py:168]: im late to the party, but theres a MycroftSkill method that handles this `self.make_active()`

Comment (ken-mycroft) [reply mycroft/skills/common_play_skill.py:168]: It was getting called inconsistently based on match types. I deprecated it in the next PR

**Pull request merged**